src/measure_theory/measure/measure_space_def.lean

```diff
@@ -192,6 +192,28 @@ lemma exists_measurable_superset_iff_measure_eq_zero :
   (∃ t, s ⊆ t ∧ measurable_set t ∧ μ t = 0) ↔ μ s = 0 :=
 ⟨λ ⟨t, hst, _, ht⟩, measure_mono_null hst ht, exists_measurable_superset_of_null⟩
 
+lemma measurable_set_preimage_iterate {f : α → α} (hf : measurable f)
```

Comment:
Could you add the `@[measurability]` tag to this lemma and the next two?
Also, they have nothing to do with a measure space specifically, do they? Could you move them to another file, like `measurable_space`?

Reply:
Good point, now done.